### PR TITLE
Brand new Dispatchers section with up-to-date information from Akka.NET.

### PR DESCRIPTION
**Pull request by nvivo**
I wrote all of the text from scratch by looking at the original akka docs and the akka.net source code.

Please [take a look](https://github.com/nvivo/getakka.net/blob/dispatchers/src/docs/working-with-actors/Dispatchers.md) before merging and see if it's ok. If there is anything I shouldn't have said or that is conceptually wrong, please let me know and I'll fix it.

Ref #7

### CODE REVIEW

File: src/docs/working-with-actors/Dispatchers.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+title: Dispatchers
+---
+
+# Dispatchers
+
+## What Dispatchers Do?
+
+Dispatchers are responsible for scheduling all code that run inside the `ActorSystem`. Dispatchers are one of the most important parts of Akka.NET, as they control the throughput and time share for each of the actors, giving each one a fair share of resources.
+
+By default, all actors share a single **Global Dispatcher**. By default, this dispatcher uses the *.NET Thread Pool* behind the scenes, which is optimized for most common scenarios. That means the default configuration should be *good enough* for most cases.
```
Comment (rogeralsing) [inline src/docs/working-with-actors/Dispatchers.md]: Two "by default" in two sentensens , rephrase one?

**Comment by rogeralsing** [comment]
Looks great!

**Comment by nvivo** [comment]
@rogeralsing, do you want me to squash?

**Comment by nvivo** [comment]
Squashed!

**Pull request merged**